src/Components/ServiceScene/ServiceScene.tsx

```diff
@@ -149,24 +139,55 @@ export class ServiceScene extends SceneObjectBase<ServiceSceneState> {
 
     this.setBreakdownView();
     this.setEmptyFiltersRedirection();
+    const slug = getDrilldownSlug();
+
+    // If we don't have a patterns count in the tabs, or we are activating the patterns scene, run the pattern query
+    if (this.state.patternsCount === undefined || slug === 'patterns') {
+      this.state.$patternsData.runQueries();
```

Comment:
This will run uncessarily when the scene has already been activated, check for data first!